<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>更纱黑体 UI SC</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hLayout1">
      <item>
       <widget class="QLabel" name="labelEdit">
        <property name="text">
         <string>编辑操作</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonAdd">
        <property name="text">
         <string>添加文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonRemove">
        <property name="text">
         <string>移除文件</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hLayout2">
      <item>
       <widget class="QLabel" name="labelFolder">
        <property name="text">
         <string>输出文件夹</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputFolder"/>
      </item>
      <item>
       <widget class="QToolButton" name="buttonFolder">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hLayout3">
      <item>
       <widget class="QLabel" name="labelDecoding">
        <property name="text">
         <string>源编码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="inputDecoding"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hLayout4">
      <item>
       <widget class="QLabel" name="labelEncoding">
        <property name="text">
         <string>目标编码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="inputEncoding"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="buttonStart">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>开始转换</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
